content/working_with/console/pages/_index.md

```diff
@@ -22,42 +22,62 @@ Below are the default set of pages (templates) presented per {{< param product_n
 ***main-sys_admin*** template applies to `sys-admin` users in all the tenants, and includes the following pages:
 
 * **Dashboard** - An overview of the existing executions and system statistics.
-* **Cloudify Catalog** - Contains Plugins Catalog showing available {{< param product_name >}} plugins and Blueprints Catalog presenting blueprint examples.
-* **Local Blueprints** - A list of all the blueprints which were uploaded to the current tenant and access to [{{< param cfy_composer_name >}}]({{< param cfy_composer_link >}}). From this page you can access the Blueprint drill-down page, by clicking on a specific blueprint.
-* **Site Management** - A list of all sites created in the current tenant and possibility to manage the sites.
-* **Deployments** - A list of all deployments created in the current tenant, and the statuses of their nodes. From this page you can access the Deployment drill-down page, by clicking on a specific deployment.
-* **Tenant Management** - Users, User-Groups and Tenants Management.
-* **Admin Operations** - Maintenance Mode switch, a view-only presentation of the system’s cluster status and Snapshots management.
-* **System Resources** - Plugins, Secrets and Agents management.
-* **Logs** - Events/Logs table with multiple filters.
-
+* **Blueprints** - A list of all the blueprints which were uploaded to the current tenant.
+The Blueprint drill-down page can be accessed by clicking a specific blueprint.
+New blueprints can be uploaded to the system by either using specific blueprint package or by selecting one of blueprint examples available in the **Marketplace**.
```

Comment:
And now we are going to have also Terraform option. Are you planning to add it here later when all TF related changes are in place? I don't see specific task for documentation update in RD-3206 epic, so I think we should create one. Or you had another plan for it?

Reply:
My plan was to cover it as part of separate ticket dedicated for terraform related changes - I created RD-3969 as indeed there was no ticket for this.